docs/migrations.rst

```diff
@@ -8,9 +8,9 @@ Migration Guide
 This page contains information about changes between major versions and
 how you can migrate from one version to another.
 
-.. _migration-to-0-15-0:
+.. _migration-to-rasa-1.0:
 
-0.14.x to 0.15.0
+0.x.x to Rasa 1.0
```

Comment:
should stay the latest, as we are not including how to get from e.g. 0.13 to 1
```suggestion
0.14.x to Rasa 1.0
```

Reply:
Gotcha. Just didn't know what to write as core is on 14 and NLU 15